src/Microsoft.Azure.WebJobs/BindingAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host.Config
+{
+    /// <summary>
+    /// Place this on an attribute to note that it's a binding attribute. 
+    /// An extension should then claim this and bind it. 
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
```

Comment:
To lightup with BYOB and VS tooling, yes. So existing scenarios aren't broken.  


---
In reply to: [110183092](https://github.com/Azure/azure-webjobs-sdk/pull/1093#discussion_r110183092) [](ancestors = 110183092)